addons/sale_stock/migrations/11.0.1.0/post-migration.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def update_procurement_field_from_sale(cr):
+    """
+    Connecting new sale_id field of procurement_group
+    with respective sale orders
+    (considering procurement_group_id of sale order)
+    """
+    query = ("UPDATE procurement_group AS pg "
+             "SET sale_id = so.id "
+             "FROM sale_order as so "
+             "WHERE so.procurement_group_id = pg.id")
+    openupgrade.logged_query(
+        cr, query, ()
+    )
+
+
+def update_stock_move_field_from_procurement_order(cr):
+    """
+    Filling the values of new sale_order_id field of stock_move
+    with respective values of old procurement_order
+    """
+    query = ("UPDATE stock_move AS sm "
+             "SET sale_order_id = po.sale_order_id "
```

Comment:
I do not have `sale_order_id` on `procurement.order` model in V10 only `sale_line_id` if i'm right ?

Reply:
you are right